examples/terrajet-aws-provider/managed-resources/kms.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
What do you think about having examples for managed resources? Crossplane's use cases pretty much expects compositions to be used. I feel like managed resources examples can be found in compositions.